mama/c_cpp/src/c/subscriptionimpl.h

```diff
@@ -221,7 +221,9 @@ mamaSubscription_getSubscriptionBridge(
  *              MAMA_STATUS_NULL_ARG
  *              MAAM_STATUS_OK
  */              
-mama_status mamaSubscription_getExpectingInitial(
+MAMAExpDLL
+extern mama_status
+mamaSubscription_getExpectingInitial(
```

Comment:
This is for my test bridge (reflect) so it can provide a mode where an initial is guaranteed to be the first msg received.